scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -1165,9 +1176,9 @@ def __init__(self, labels_as_attributes=False, label_prefix='', label_include_gl
         """
         @param: labels_as_attributes - boolean - if True any labels that are not excluded will be added to the attributes result dict
```

Comment:
Sorry, i didn't notice the types being slipped in here the first time.  Just move the parameter types to `@type` tags.

Reply:
Done.  Will pay more attention to this going forward.